### DISCUSSION TITLE
Faucet through API

### DISCUSSION THREAD

**Discussion by syedzahidulla** [0xPolygon/matic.js#326] [Q&A]
Hi Iam trying to build an app where intially for developement purpose i need to use MATIC Token through faucet.

I want to know is there any api to do that programatically.

Because it is not feasable for me evertime to goto faucet website and do it manually.

Can somebody help me with that?

**Comment by ujjwalgupta94** [comment]
faucet is supposed to help new users to do transaction and get started into polygon network. It's not free - behind the scene polygon pays for it. 

So it's not possible to get the token through faucet using program. The best way is to buy matic from some exchange and use it.
